oaknational/Oak-Web-Application
# fix: wrap quiz hints in mathjax

## Bug Description

Quiz questions where  hints have a formula  eg. https://www.thenational.academy/pupils/programmes/maths-secondary-year-7/units/expressions-and-equations/lessons/highest-common-factor-with-algebraic-terms/exit-quiz
are not being rendered with MathJax

## Fixes

- Wrapped the hint in MathJax

## How to test

1. Navigate to  https://deploy-preview-2667--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-secondary-year-7/units/expressions-and-equations/lessons/highest-common-factor-with-algebraic-terms/exit-quiz
2. Click ‘Need a hint?’
3. Observe that formulae is now rendered correctly

## Screenshots

How it used to look (delete if n/a):
![image](https://github.com/user-attachments/assets/706f4c91-1caa-4b26-b280-f7e16332b78f)


How it should now look:
![image](https://github.com/user-attachments/assets/30feac35-8a2f-4a87-ab5f-5541347dec1e)



### ACs

- [x]  hints with formulae correctly rendered
- [x]  other hints unaffected
